src/core/qgsvectorlayerlabeling.h

```diff
@@ -87,6 +87,17 @@ class CORE_EXPORT QgsAbstractVectorLayerLabeling
       parent.appendChild( doc.createComment( QStringLiteral( "SE Export for %1 not implemented yet" ).arg( type() ) ) );
     }
 
+  protected:
+
+    /**
+     * Writes a TextSymbolizer element contents based on the provided labeling settings
+     * @brief writeTextSymbolizer
+     * @param parent
+     * @param settings
+     * @param props
+     */
```

Comment:
Can you remove all those empty `@param` and `@brief` (and document the ones that are important for this implementation)

Reply:
Adding doc for all of them